crates/viewer/re_viewer/src/web.rs

```diff
@@ -236,8 +247,12 @@ impl WebHandle {
             return;
         };
 
-        if let Some(tx) = self.tx_channels.remove(id) {
-            tx.quit(None).warn_on_err_once("Failed to send quit marker");
+        if let Some(channel) = self.tx_channels.remove(id) {
+            channel
+                .log_tx
+                .quit(None)
+                .warn_on_err_once("Failed to send quit marker");
+            drop(channel.table_tx);
```

Comment:
I think we should use the destructor pattern here so we remember to update the code if we add more fields things to the `channel` type. It could also be made a member function